packages/terra-date-picker/src/terra-dev-site/doc/date-picker/CalendarFilter.3.doc.mdx

```diff
@@ -0,0 +1,64 @@
+import { Badge } from 'terra-date-picker/package.json?dev-site-package';
+
+import CalendarFilterPropsTable from 'terra-date-picker/src/CalendarFilter?dev-site-props-table';
+
+import CalendarFilterDefault from '../example/CalendarFilter/CalendarFilterDefault?dev-site-example';
+import CalendarFilterExcludeDates from '../example/CalendarFilter/CalendarFilterExcludeDates?dev-site-example';
+import CalendarFilterIncludeDates from '../example/CalendarFilter/CalendarFilterIncludeDates?dev-site-example';
+import CalendarFilterFilterDates from '../example/CalendarFilter/CalendarFilterFilterDates?dev-site-example';
+import CalendarFilterMinMax from '../example/CalendarFilter/CalendarFilterMinMax?dev-site-example';
+import CalendarFilterDefaultDate from '../example/CalendarFilter/CalendarFilterDefaultDate?dev-site-example';
+import CalendarFilterControlled from '../example/CalendarFilter/CalendarFilterControlled?dev-site-example';
+
+<Badge />
+
+# Calendar Filter
+
+CalendarFilter is a sub-component of [terra-date-picker](https://github.com/cerner/terra-framework/tree/master/packages/terra-date-picker) component which renders a faceup calendar for selecting a date.
+
+Unlike the DatePicker which is used to pick a day to return to submit to a form, the CalendarFilter would be useful in applications where the calendar is intended to be used as an interactive filter that changes some alternate dependent UI item. An example of this is a scheduling application where a selecting a day in the CalendarFilter can lead to displaying a list of appointments.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-date-picker`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+| react-intl | ^2.8.0 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Implementation Notes:
+CalendarFilter is required to be composed inside the [Base](https://engineering.cerner.com/terra-core/components/terra-base/base/base) component with locale in order for it to load the correct date format and translation strings.
```

Comment:
We should just get rid of this note (also [here](https://github.com/cerner/terra-framework/blame/Inline-Date-Picker/packages/terra-date-picker/src/terra-dev-site/doc/date-picker/DatePicker.1.doc.mdx#L75) and [here](https://github.com/cerner/terra-framework/blame/Inline-Date-Picker/packages/terra-date-picker/src/terra-dev-site/doc/date-picker/DatePickerField.2.doc.mdx#L19)) since terra-base will be deprecated soon

Reply:
Sure, removed from Docs : [35047bbc5a9](https://github.com/cerner/terra-framework/pull/1134/commits/e50ba975191da70534cc031477e27ce3569f9145#diff-f5fd65fbc8ca01b80b62c35047bbc5a9)